{"name": "Verify that the order id is not returned if passing invalid track number.", "status": "passed", "description": "Verify that 404 code is returned for GET request if passing invalid track number.", "steps": [{"name": "Get order id.", "status": "passed", "parameters": [{"name": "track_number", "value": "1"}], "start": 1742386046546, "stop": 1742386046966}], "start": 1742386046546, "stop": 1742386046967, "uuid": "77c80eb4-c38c-43c2-8486-0a7f943a7b39", "historyId": "2ad7e1eb0519dabe786fa4e02c4915f8", "testCaseId": "2ad7e1eb0519dabe786fa4e02c4915f8", "fullName": "tests.test_order_get_by_id.TestOrderGetByID#test_get_order_id_by_invalid_track_number_not_returned", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_order_get_by_id"}, {"name": "subSuite", "value": "TestOrderGetByID"}, {"name": "host", "value": "ip-192-168-100-6.eu-north-1.compute.internal"}, {"name": "thread", "value": "20517-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_order_get_by_id"}]}